src/saml.ts

```diff
@@ -158,6 +158,7 @@ class SAML {
       allowCreate: ctorOptions.allowCreate ?? true,
       spNameQualifier: ctorOptions.spNameQualifier,
       wantAssertionsSigned: ctorOptions.wantAssertionsSigned ?? false,
+      wantDocumentSigned: ctorOptions.wantDocumentSigned ?? false,
```

Comment:
About naming...this is about signed authentication response. `Document` is quite generic term. SP may receive also `LogoutRequest` and `LogoutResponse` XML/SAML documents. Could it be `wantAuthenticationResponseSigned`?

Having said that I must also note that e.g. [tngan/samlify](https://github.com/tngan/samlify) seems to use `wantMessageSigned` (reference [tngan/samlify v2.8.4 docs/signed-saml-response.md](https://github.com/tngan/samlify/blob/9953b333ba2ab1613b671d0c2322db561895b3fd/docs/signed-saml-response.md))

Reply:
I appreciate that this name is poor. I'm trying to find some SAML documentation to support having this option and see what it says. The closest that I've found is `WantAuthnRequestsSigned` and `authnRequestsSigned`, which doesn't appear to be the same.

Reply:
I agree a  good explicit name adds value, something like `wantAuthResponseSigned`.